src/lib.rs

```diff
@@ -58,6 +59,24 @@ use sync::atomic::AtomicUsize;
 use sync::atomic::Ordering::SeqCst;
 use sync::{Condvar, Mutex};
 
+#[cfg(coverage)]
+#[coverage(off)]
+fn cov_assert(x: bool, msg: &str) {
```

Comment:
Personally, I'm not in favor of complicating our code to avoid upstream issues on interactions of coverage and macro.